packages/@react-aria/focus/src/FocusScope.tsx

```diff
@@ -36,6 +36,8 @@ interface FocusManager {
 const FocusContext = React.createContext<FocusManager>(null);
 
 let activeScope: RefObject<HTMLElement[]> = null;
+let counter = 0;
+let scopesMap: Map<number, RefObject<HTMLElement[]>> = new Map();
```

Comment:
You could use a Set for this instead of a Map since it doesn't look like you're using the keys anywhere. Then you could get rid of the counter too.